generate_docs.py

```diff
@@ -9,6 +9,13 @@
 
 MD_ANCHOR_LINKS = r"\[(.+)\]\(#.+\)"
 
+def find_file(directory, filename):
```

Comment:
It feels like this should return the path to the file rather than the filename but it's a small nit